src/components/Integrations/BigQuery/ControlPanel/Input.tsx

```diff
@@ -25,9 +25,8 @@ const Input = (props: InputProps) => {
   const ButtonRemove = useMemo(() => {
     if (className === 'style_not_empty') {
       return (
-        <button type="button" onClick={handleRemove}>
-          {' '}
-          -{' '}
+        <button type="button" onClick={handleRemove} data-testid="btn-remove">
```

Comment:
why `data-testid` attribute? you can react elements with `getByText` in the tests, or you can container.querySelector